website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

```diff
@@ -102,27 +102,27 @@ The way you set up and configure a new policy set depends on your workflow and w
    - For version control policies, choose a version control provider and then select the repository with your policy set. Terraform Cloud shows a form to create a policy set connected to that repository.
    - For automated workflows, click **No VCS Connection**. Terraform Cloud shows a form to create an empty policy set. You can use the API to add policies to this empty policy set later.
-1. Configure the policy set settings:
-   - **Policy Framework:**  Choose the policy framework for the policies you want to add. You can only create a policy set from policies written using the same framework. You cannot change the framework type after creation.
-   - **Name:** Add a name containing letters, numbers, `-`, and `_`. Terraform Cloud displays this name in the UI. The name must be unique to your organization.
-   - **Description:** Describe the policy set’s purpose. The description supports Markdown rendering, and Terraform Cloud displays this text in the UI.
-   - **Scope of policies:** Choose whether Terraform Cloud should automatically enforce the policy set on all workspaces, or only on a specific subset.
-   - **(Optional) Workspaces**: A **Workspaces** section appears on the bottom of the form when you scope the policy set to selected workspaces. Select workspaces where Terraform Cloud should apply the policy set.
-   - **(OPA Only) Overrides**:  Choose whether users with override policy permissions can let Terraform apply plans that have [mandatory policy](#policy-enforcement-levels) failures.
-   - **(VCS Only) VCS Branch**: Specify the branch within your VCS repository where Terraform Cloud should import new versions of policies. If you do not set this field, Terraform Cloud uses the default branch of the VCS repository you selected.
-   - **(VCS Only) Policies Path**: Specify the sub-directory in your VCS repository containing the policy set files. This action lets you maintain multiple policy sets within a single repository. Set this field to the directory path that contains the `sentinel.hcl` (Sentinel) or `policies.hcl` (OPA) configuration file for the policy set. You can start the path with `/`, but this is optional. Terraform Cloud assumes that relative paths originate from the root of the repository. If you do not set this field, Terraform Cloud uses the repository root directory.
-   - **(Managed Policies Only) Policies**: Select managed policies to add to the policy set. You can only add policies written with the policy framework you selected for the policy set.
+1. Choose a **Policy framework** for the policies you want to add. You can only create a policy set from policies written using the same framework. You cannot change the framework type after creation.
+1. Choose a policy set scope:
+   - **Policies enforced globally:** Terraform Cloud will automatically enforce this global policy set on all existing and future workspaces.
```

Comment:
on all existing and future workspaces + and projects?

Reply:
Yes, it is enforced on all the workspaces in the organization.
@ker-an Maybe we can add "in the organization" to be more explicit?

Reply:
Yup! Great suggestion @hs26gill, I will add that bit.